indicators/forms.py

```diff
@@ -295,6 +301,12 @@ def get_helptext(disagg):
                     _('This disaggregation cannot be unselected, because it was already used in submitted program results.')
                 )
             return helptext
+        # this is duplicated because of the wacky way we are handling disaggregations in the modal form
+        # if you need to update this, also update it in the indicator model
+        disaggregation_group_helptext = _("Select all relevant disaggregations. Disaggregations are managed by the "
```

Comment:
Translator comment?  Maybe fine without, since it's a full sentence, but might be nice to give them some context.